src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OrganisationsController.cs

```diff
@@ -84,5 +84,25 @@ public async Task<ActionResult> GetByIdAsync(Guid id)
                 }
             });
         }
+
+        [HttpPut]
+        [Route("{id}")]
+        public async Task<ActionResult> UpdateOrganisationByIdAsync(Guid id, UpdateOrganisationViewModel viewModel)
+        {
+            if (viewModel is null)
+                throw new ArgumentNullException(nameof(viewModel));
```

Comment:
Might be worth sticking to what we have done throught and use { } for if statements

Reply:
I think this was raised in the code standards meeting but we didn't reach a conclusion. 
Although I prefer 1 line statements this way, I don't mind compromising here. I'll add the braces :]

Reply:
The code standards decision was developer's prerogative on this one.

Reply:
I remember wrongly then 😄
I'm not that fussed about this, plus, I've already added the brackets in.